xbmc/rendering/RenderSystem.cpp

```diff
@@ -57,7 +58,10 @@ bool CRenderSystemBase::SupportsStereo(RENDER_STEREO_MODE mode) const
 
 void CRenderSystemBase::ShowSplash(const std::string& message)
 {
-  if (!CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_splashImage && !(m_splashImage || !message.empty()))
+  if ((!CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
+           CSettings::SETTING_LOOKANDFEEL_SHOWSPLASH) ||
+       !CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_splashImage) &&
```

Comment:
the advancedsetting should probably take precedence here

Reply:
i would remove the as if the gui setting is there. what do you think?

Reply:
@wsnipex I change the order so that it checks advancedsetting first.

@howie-f I think it's better to keep both configuration scenarios, someone will prefer to continue as usual and others prefer the GUI.